### DISCUSSION TITLE
Logging with home assistant.

### DISCUSSION THREAD

**Discussion by Herbs851** [zwave-js/zwave-js#3113] [Q&A] [answered]
Hi all,

I am trying to install zwavejs in a python venv with nodeenv.
I know this is not the recommended method but everything seems to work fine ;)

Just a detail when in zwave_config.json I implement these directives :

![image](https://user-images.githubusercontent.com/51154851/127486860-07b078d4-c640-443e-bf76-f003222189b0.png)

The logs are written to the right file until I use the home assistant ui (core installation) to display the zwavjs logs.

From then, the logs are written to the browser and are no longer written to the log file.

![zwavejs](https://user-images.githubusercontent.com/51154851/127487711-935903e5-a0e9-4acb-b9b8-307c16251555.gif)

and after displaying logs with home assistan ui :

![zwavejs2](https://user-images.githubusercontent.com/51154851/127487811-3c7910d4-a13a-48bd-8b49-e7efcaf91241.gif)

Do you have any advice on how to make the 2 work at the same time ?

Maybe this deal with the zwavejs integration of home assistant but thank you in advance !!!

Bye.

**Comment by AlCalzone** [comment, marked as answer]
I suspect this has to do with how HA configures the logging. I'll notify the devs about this, there's something else that's not working correctly.

**Reply by Herbs851** [reply]
Ok perfect.

Thanks a lot !!!
